managing-os/templates/fedora-minimal.md

```diff
@@ -63,9 +63,21 @@ Use case | Description | Required steps
 **NetVM (extra firmware)** | If your network devices need extra packages for the template to work as a network VM | Use the `lspci` command to identify the devices, then run `dnf search firmware` (replace `firmware` with the appropriate device identifier) to find the needed packages and then install them.
 **Network utilities** | If you need utilities for debugging and analyzing network connections | Install the following packages: `tcpdump` `telnet` `nmap` `nmap-ncat`
 **USB** | If you want USB input forwarding to use this template as the basis for a [USB](/doc/usb/) qube such as `sys-usb` | Install `qubes-input-proxy-sender`
-**VPN** | You can use this template as basis for a [VPN](/doc/vpn/) qube | Use the `dnf search "NetworkManager VPN plugin"` command to look up the VPN packages you need, based on the VPN technology you'll be using, and install them. Some GNOME related packages may be needed as well. After creation of a machine based on this template, follow the [VPN howto](/doc/vpn/#set-up-a-proxyvm-as-a-vpn-gateway-using-networkmanager) to configure it.
+**VPN** | You can use this template as basis for a [VPN](/doc/vpn/) qube | Use the `dnf search "NetworkManager VPN plugin"` command to look up the VPN packages you need, based on the VPN technology you'll be using, and install them. GNOME related packages are needed as well in order to edit your VPN connections through the Network Applet. After creation of a machine based on this template, follow the [VPN howto](/doc/vpn/#set-up-a-proxyvm-as-a-vpn-gateway-using-networkmanager) to configure it.
+**3G Modems**| If you want to use this template as a NetVM using a 3G Modem connection| Install the following packages: `ModemManager` `NetworkManager-wwan` `usb_modeswitch`. Additionally, `modem-manager-gui` can help troubleshooting your 3g modem [1].
```

Comment:
Not sure about actual github pages, but github interface is confused about formatting, because of missing space before `|`.

Reply:
It looks OK with Jekyll.